scripts/dev/contexts/variables/om60

```diff
@@ -2,12 +2,9 @@
 
 set -Eeou pipefail
 
-script_name=$(readlink -f "${BASH_SOURCE[0]}")
-script_dir=$(dirname "${script_name}")
-
 CUSTOM_OM_PREV_VERSION=6.0.0
 export CUSTOM_OM_PREV_VERSION
-CUSTOM_OM_VERSION=$(grep -E "^\s*-\s*&ops_manager_60_latest\s+(\S+)\s+#" <"${script_dir}"/../../../../.evergreen.yml | awk '{print $3}')
+CUSTOM_OM_VERSION=$(grep -E "^\s*-\s*&ops_manager_60_latest\s+(\S+)\s+#" <"${PROJECT_DIR}"/.evergreen.yml | awk '{print $3}')
```

Comment:
Not something for this PR, but I think we should switch for yq/jq for reading yaml/json instead of awk.